tracer/src/Datadog.Trace/RuntimeMetrics/RuntimeMetricsWriter.cs

```diff
@@ -179,17 +182,30 @@ internal void PushEvents()
                     Log.Debug("Sent the following metrics to the DD agent: {Metrics}", ProcessMetrics);
                 }
 
+                bool sentExceptionCount = false;
+
+                if (Volatile.Read(ref _outOfMemoryCount) > 0)
+                {
+                    var oomCount = Interlocked.Exchange(ref _outOfMemoryCount, 0);
```

Comment:
Is there not a race condition here? 🤔 I guess it's the sort we don't care about too much🤷‍♂️

Reply:
Yeah, worst case scenario we push the points next time the timer ticks